packages/fxa-auth-server/test/remote/account_reset_tests.js

```diff
@@ -21,239 +22,194 @@ describe('remote account reset', function() {
     });
   });
 
-  it('account reset w/o sessionToken', () => {
+  it('account reset w/o sessionToken', async () => {
```

Comment:
I async-awaitified these tests while I was in here, but I think in retrospect it generated a bit much churn. Sorry for the added noise to this PR, I refrained from doing any of the other test files.